extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/exec/WorkerImpl.java

```diff
@@ -582,12 +588,14 @@ public void postCleanupStage(final StageId stageId)
   @Override
   public void postFinish()
   {
+    log.info("Finish received");
     kernelManipulationQueue.add(KernelHolder::setDone);
   }
 
   @Override
   public ClusterByStatisticsSnapshot fetchStatisticsSnapshot(StageId stageId)
   {
+    log.info("Fetching statistics for stage:[%d]", stageId.getStageNumber());
```

Comment:
As above. Inconsistent use of colon. No colon is needed here: `stage %d`.

Does the logger automagically fill in the task ID? If not, would be good to include that so we can find events for a specific task.

Reply:
Each worker can only have one taskID and each log is generally task log is generally pulled from the druid console. 
In the case, people are using popular log agg tools, then they should add a dimension taskId to the "log line" via the constructs provided by the tool, "looking at file name in case of druid to figure out the taskId" is one such approach.
Hence I am not pushing taskId everywhere which just bloats the message.